ibis/file/csv.py

```diff
@@ -98,10 +99,10 @@ def csv_read_table(op, client, scope, **kwargs):
 
 
 @pre_execute.register(ops.Selection, CSVClient)
-def csv_pre_execute_selection(op, client, scope, **kwargs):
+def csv_pre_execute_selection(op, client, scope, timecontext=None, **kwargs):
     tables = filter(lambda t: t not in scope, physical_tables(op.table.op()))
 
-    ops = {}
+    ops = Scope({})
```

Comment:
Can you do `ops = Scope()` instead here?

Reply:
changed the constructor a little bit. Now empty scope could be created without any param specified.